pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "movie-surfer"
+version = "0.1.0"
+description = ""
+authors = ["Islam Iangurazov <islamyang123@gmail.com>"]
+readme = "README.md"
+packages = [{include = "movie_surfer"}]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+click = "^8.1.3"
+sphinx = "^7.0.0"
```

Comment:
we only need it to build documentation
it probably make sense to make another group of dependencies and place it there
what do you think?

Reply:
agreed.